docs/reference/modules/snapshots.asciidoc

```diff
@@ -121,6 +121,7 @@ The following settings are supported:
  using size value notation, i.e. 1g, 10m, 5k. Defaults to `null` (unlimited chunk size).
 `max_restore_bytes_per_sec`:: Throttles per node restore rate. Defaults to `40mb` per second.
 `max_snapshot_bytes_per_sec`:: Throttles per node snapshot rate. Defaults to `40mb` per second.
+`readonly`:: Makes repository read-only. Defaults to `false`.
```

Comment:
The only concern I have is about the documentation but it's not exactly related to this change.
For example, the new option `readonly` only appears within Shared FS repository although it will be a "common" option to many repositories.

Should we rewrite this part of the doc?